src/Common/Config/configReadClient.cpp

```diff
@@ -10,14 +10,25 @@ namespace DB
 {
 bool configReadClient(Poco::Util::LayeredConfiguration & config, const std::string & home_path)
 {
+    auto safe_exists = [](const auto & path)
+    {
+        std::error_code ec;
+        bool res = fs::exists(path, ec);
+        if (ec.value() > 0)
+        {
+            LOG_ERROR(&Poco::Logger::get("ClickHouseClient"), "Can't check '{}': [{}]({})", path, ec.value(), ec.message());
```

Comment:
AFAIK clickhouse-client does not use logger, it is using std::cerr instead.

Reply:
Nevertheless, if the message is printed, it is fine.

Reply:
Fixed